tests/bsim/bluetooth/mesh/src/test_transport.c

```diff
@@ -168,6 +168,32 @@ static void test_tx_group(void)
 	PASS();
 }
 
+/** Test sending to a fixed group address. */
+static void test_tx_fixed(void)
+{
+	int err;
+
+	bt_mesh_test_setup();
+
+	err = bt_mesh_test_send(BT_MESH_ADDR_RELAYS, NULL, test_vector[0].len,
```

Comment:
There is a special macro to check the result that the function returns `ASSERT_OK`.
I'd recommend using it. 
```
ASSERT_OK(bt_mesh_test_send(BT_MESH_ADDR_RELAYS, NULL, test_vector[0].len,test_vector[0].flags, K_SECONDS(2)));
```